virttest/virt_vm.py

```diff
@@ -1216,6 +1216,8 @@ def print_guest_network_info():
                 if serial:
                     break
                 raise
+            except remote.LoginProcessTerminatedError:
```

Comment:
Can this additional exception:remote.LoginProcessTerminatedError  combined with previous  remote.LoginAuthenticationError ?  In previous LoginAuthenticationError: if serial : break, otherwise  raise

Reply:
if serial is true, in the while loop of wait_for_serial_login, lots of logs will still be printed out. And when the process is terminated, even serial is true, the code still can do nothing to make a success result.
I don't think it's a good idea to put together with the LoginAuthenticationError exceptions.
@chunfuwen

Reply:
this is heavily common used function, please paste your test result once ready